dask_cuda/utils_test.py

```diff
@@ -0,0 +1,28 @@
+import os
+from zict.file import _safe_key as safe_key
+import string
+from random import choice
+
+
+def assert_device_host_file_size(dhf, total_bytes, chunk_overhead=1024):
```

Comment:
If this is only used in the one testing file then I recommend that we keep it there for now.

Reply:
Or perhaps it should be a `_validate` method on the `DeviceHostFile` object itself?

Reply:
> Or perhaps it should be a `_validate` method on the `DeviceHostFile` object itself?

I don't think so. I'm using this solely for testing and the 1024 bytes is simply a safe size. As far as I could understand, the actual size may vary depending on the actual type being serialized, as well as through how many serialization stages it has passed. In other words, it's difficult to determine how much this overhead will be, and for that reason, I don't think it's a class-worthy method.

Reply:
OK. OK to move it to the testing file then?  If it's only likely to be used there then I'd prefer to keep it local to there.

Reply:
Done.